core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -122,6 +123,8 @@ public final class VirtualHostBuilder implements TlsSetters {
     private String hostnamePattern;
     private int port = -1;
     @Nullable
+    private String contextPath;
```

Comment:
Can we just set to `/` by default and remove the `@Nullable` contract?

Reply:
I'll try to address https://github.com/line/armeria/pull/4802#discussion_r1267647512 comment first!